lib/tasks/cucumber.rake

```diff
@@ -51,21 +51,21 @@ begin
     end
   end
   desc 'Alias for cucumber:ok'
-  task :cucumber => 'cucumber:ok'
+  task cucumber: 'cucumber:ok'
 
-  task :default => :cucumber
+  task default: :cucumber
```

Comment:
have you tried to run cucumber after changing this?

Reply:
yes.